libbeat/logp/core_test.go

```diff
@@ -116,3 +118,14 @@ func TestL(t *testing.T) {
 		assert.Equal(t, "warning 1", log.Message)
 	}
 }
+
+func TestAllStdoutDisablesDefaultGoLogger(t *testing.T) {
+	DevelopmentSetup(WithSelectors("*"))
+	assert.Equal(t, ioutil.Discard, golog.Writer())
+
+	DevelopmentSetup(WithSelectors("stdlog"))
+	assert.Equal(t, ioutil.Discard, golog.Writer())
+
+	DevelopmentSetup(WithSelectors("*", "stdlog"))
+	assert.NotEqual(t, ioutil.Discard, golog.Writer())
```

Comment:
I'd like to see a test case with a different log level and a test case with no selectors.

Reply:
based on the above log level suggestion and selectors https://github.com/elastic/beats/pull/15708/files#r370756892